sql/updates/world/master/xxxx_xx_xx_x1_world.sql

```diff
@@ -0,0 +1,97 @@
+UPDATE `trinity_string` SET `content_default`='Up to %i expansions are allowed now.' WHERE  `entry`=61;
```

Comment:
" I hope these commits are not merged together. This is because of the first being a cherry pick of 3.3.5 branch commit and the second is more work built on top of that."

Still want it though?